packages/docs/src/routes/api/qwik/index.md

```diff
@@ -10647,7 +10647,9 @@ This method works like an async memoized function that runs whenever some tracke
 
 The status can be one of the following:
 
-- 'pending' - the data is not yet available. - 'resolved' - the data is available. - 'rejected' - the data is not available due to an error or timeout.
+- `pending` - the data is not yet available. - `resolved` - the data is available. - `rejected` - the data is not available due to an error or timeout.
```

Comment:
this is likely included because your fork might be outdated from main

Reply:
Ah, actually, I synced it, but forgot to pull changes to my local machine. Lemme fix this :)

Reply:
I rebased it

Reply:
no worries

Reply:
Or you can do this :)